entrypoint.sh

```diff
@@ -167,6 +205,10 @@ function lint_files() {
 	local files_to_lint=""
 	local env_var_name="${linter_array[env]}"
 
+	if [ ${RE_RUN:-false} == "true" ] && has_autofix "${linter_array[name]}"; then
```

Comment:
Based on this, I think `RE_RUN` means "this is a re_run" but it's still a little bit of a guess for me.  I think what would be clearer is calling the var something like `LINT_ROUND` or `ITERATION` and set it to 1 for the first time and 2 for the re run.  Or something like that so it's readable/understandable

Reply:
Will do